### ISSUE TITLE
Update to 1.6.0.1: TypeError: Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by joshuadavidson** [meteor/meteor#9476]
### OS
Windows 10 Pro, Version 1709, 64-bit.

### Issue
Currently have a 1.4.4.4 app that I attempted to update to 1.6.0.1 today. I started by uninstalling my previous versions of meteor and installing the new 64bit version using `choco install meteor -y`. I then successfully ran `meteor update --release 1.6.0.1`. I then ran `meteor reset`. When I go to run my app with `meteor run --settings=settings-local.json` I get the following error:
```
C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\fibers\future.js:280
                                                throw(ex);
                                                ^

TypeError: Cannot read property '0' of undefined
    at toDosPath (C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\tools\static-assets\server\mini-files.js:41:8)
    at C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\tools\static-assets\server\mini-files.js:98:16
    at Array.map (<anonymous>)
    at Function._.map._.collect (C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:97:56)
    at C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\tools\static-assets\server\mini-files.js:95:16
    at ImportScanner._getRelativeImportId (C:\tools\isobuild\import-scanner.js:616:43)
    at each (C:\tools\isobuild\import-scanner.js:600:33)
    at _.each._.forEach (C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._resolve (C:\tools\isobuild\import-scanner.js:589:7)
    at each (C:\tools\isobuild\import-scanner.js:682:29)
    at _.each._.forEach (C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._scanFile (C:\tools\isobuild\import-scanner.js:673:5)
    at each (C:\tools\isobuild\import-scanner.js:763:12)
    at _.each._.forEach (C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._scanFile (C:\tools\isobuild\import-scanner.js:673:5)
    at each (C:\tools\isobuild\import-scanner.js:763:12)
    at _.each._.forEach (C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._scanFile (C:\tools\isobuild\import-scanner.js:673:5)
    at each (C:\tools\isobuild\import-scanner.js:763:12)
    at _.each._.forEach (C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._scanFile (C:\tools\isobuild\import-scanner.js:673:5)
    at each (C:\tools\isobuild\import-scanner.js:763:12)
    at _.each._.forEach (C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:87:22)
    at ImportScanner._scanFile (C:\tools\isobuild\import-scanner.js:673:5)
    at outputFiles.forEach.file (C:\tools\isobuild\import-scanner.js:382:14)
    at Array.forEach (<anonymous>)
    at ImportScanner.scanImports (C:\tools\isobuild\import-scanner.js:380:22)
    at sourceBatches.forEach.batch (C:\tools\isobuild\compiler-plugin.js:1043:17)
    at Array.forEach (<anonymous>)
    at Function.computeJsOutputFilesMap (C:\tools\isobuild\compiler-plugin.js:1011:19)
    at ClientTarget._emitResources (C:\tools\isobuild\bundler.js:1056:8)
    at buildmessage.enterJob (C:\tools\isobuild\bundler.js:827:12)
    at C:\tools\utils\buildmessage.js:359:18
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:352:34
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:350:23
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (C:\tools\utils\buildmessage.js:324:26)
    at ClientTarget.make (C:\tools\isobuild\bundler.js:818:18)
    at C:\tools\isobuild\bundler.js:2926:14
    at C:\tools\isobuild\bundler.js:3015:20
    at Array.forEach (<anonymous>)
    at Function._.each._.forEach (C:\Users\JoshDavidson\AppData\Local\.meteor\packages\meteor-tool\1.6.0_1\mt-os.windows.x86_64\dev_bundle\lib\node_modules\underscore\underscore.js:79:11)
    at C:\tools\isobuild\bundler.js:3014:7
    at C:\tools\utils\buildmessage.js:271:13
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:264:29
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:262:18
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at C:\tools\utils\buildmessage.js:253:23
    at exports.EnvironmentVariable.withValue (C:\tools\utils\fiber-helpers.js:89:14)
    at Object.capture (C:\tools\utils\buildmessage.js:252:19)
    at bundle (C:\tools\isobuild\bundler.js:2907:31)
    at files.withCache (C:\tools\isobuild\bundler.js:2854:32)
    at Object.withCache (C:\tools\fs\files.js:1658:12)
    at Object.exports.bundle (C:\tools\isobuild\bundler.js:2854:16)
    at Profile.run (C:\tools\runners\run-app.js:579:36)
    at Function.run (C:\tools\tool-env\profile.js:490:12)
    at bundleApp (C:\tools\runners\run-app.js:578:34)
    at AppRunner._runOnce (C:\tools\runners\run-app.js:622:35)
    at AppRunner._fiber (C:\tools\runners\run-app.js:880:28)
    at C:\tools\runners\run-app.js:408:12
```
I got this exact same (or wildly similar) error back when I tried to move to 1.5 (see #8740) and it turned out to be a space in my path. I learned my lesson back then and have no spaces now. This issue also seems related to #8840.

It is worth noting that I also created a test-app using `meteor create test-app` and ran it successfully. Let me know if more information is needed to help troubleshoot.

**Comment by joshuadavidson** [comment]
Just tried running the updated app using the 32-bit, 1.6.0.1 version (via `choco uninstall meteor` then `choco install meteor -y --x86`) and got the same error. Also tried removing the `.meteor/local` folder then attempting to run--didn't help.

**Comment by alanning** [comment]
I think this is the same issue as reported here:  https://github.com/meteor/meteor/issues/9333

Looks like an upstream issue with npm.  Luckily this [comment](https://github.com/meteor/meteor/issues/9333#issuecomment-346037052) on the other thread says a fix is complete and coming soon:  https://github.com/npm/npm/issues/17858#issuecomment-344522435

**Comment by hwillson** [comment]
This was fixed in npm 5.6.0, which is included with Meteor 1.6.1. Closing - thanks!
